.github/workflows/pylint.yml

```diff
@@ -0,0 +1,35 @@
+name: Linters
+
+on: [push]
+
+env:
+  POETRY_VERSION: 1.0.3
+  WORK_DIR: .
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+
+      - name: Install Pylama
+        run: |
+          python -m pip install pylama
+
+      - name: Run pylama
+        run: |
+          pylama
+
+      - name: Install Black
+        run: |
+          python -m pip install black
+
+      - name: Run black
+        run: |
+          black .
```

Comment:
Is it possible to add `isort` as a check?

Reply:
Done